docker/iml-manager-cli.dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM rust:1.39 as builder
+WORKDIR /build
+COPY . .
+RUN cd iml-manager-cli && cargo build --release
+
+FROM rust:1.39
```

Comment:
The actually final image doesn't need rust in it, as it's just running a binary at this point.